### ISSUE TITLE
MS Planner table column class changed

### ISSUE THREAD

**Issue by rempel** [drabiger/estimationsChromePlugin#8]
Fix
```
// const columns = document.querySelectorAll(".PlannerRoot .planTaskBoard .taskBoardColumn");
const columns = document.querySelectorAll(".PlannerRoot .planTaskBoardPage .taskBoardColumn");
```

**Comment by drabiger** [comment]
Thanks.

**Comment by drabiger** [comment]
Will be fixed with version 1.65 of the plugin. Fasted way to get the version is to uninstall the extension, then install it again.
